common/azure/src/main/kotlin/com/cosmotech/api/azure/adx/AzureDataExplorerClient.kt

```diff
@@ -285,6 +298,77 @@ class AzureDataExplorerClient(
     }
   }
 
+  fun ingestScenarioValidationStatus(
+      organizationId: String,
+      workspaceKey: String,
+      scenarioId: String,
+      scenarioStatus: String
+  ) {
+    val ingestionUri = csmPlatformProperties.azure?.dataWarehouseCluster?.options?.ingestionUri
+
+    val csb =
+        ConnectionStringBuilder.createWithAadApplicationCredentials(
+            ingestionUri,
+            csmPlatformProperties.azure!!.credentials.core.clientId,
+            csmPlatformProperties.azure!!.credentials.core.clientSecret,
+            csmPlatformProperties.azure!!.credentials.core.tenantId)
+
+    val ingestClient = IngestClientFactory.createClient(csb)
+    val databaseName = getDatabaseName(organizationId, workspaceKey.replace("\\s".toRegex(), ""))
+    val tableName = "ScenarioValidationStatus"
+    val ingestionProperties = IngestionProperties(databaseName, tableName)
+
+    // Creating the data mapping
+    val scenarioIdColumn = ColumnMapping("ScenarioId", "string")
+    val statusColumn = ColumnMapping("Status", "string")
+    val modifiedDateColumn = ColumnMapping("ModifiedDate", "string")
+    scenarioIdColumn.setOrdinal(0)
+    statusColumn.setOrdinal(1)
+    modifiedDateColumn.setOrdinal(2)
+
+    var columnMappings = mutableListOf<ColumnMapping>()
+    columnMappings.add(scenarioIdColumn)
+    columnMappings.add(statusColumn)
+    columnMappings.add(modifiedDateColumn)
+
+    val ingestionMapping = IngestionMapping(columnMappings.toTypedArray(), IngestionMappingKind.CSV)
+    ingestionProperties.ingestionMapping = ingestionMapping
+
+    ingestionProperties.setAdditionalProperties(mutableMapOf("ignoreFirstRecord" to "true"))
+    val data = "${scenarioId},${scenarioStatus},${ZonedDateTime.now().toLocalDate().toString()}"
+    val inputStream: InputStream = ByteArrayInputStream(StandardCharsets.UTF_8.encode(data).array())
+    val streamSourceInfo = StreamSourceInfo(inputStream)
+    ingestionProperties.dataFormat = IngestionProperties.DataFormat.CSV
+    ingestFromStream(streamSourceInfo, ingestionProperties, ingestClient)
+  }
+
+  private fun ingestFromStream(
```

Comment:
Could you please add simple test in `AzureDataExplorerClientTests` for this method ?